skimage/measure/_marching_cubes.py

```diff
@@ -101,15 +109,19 @@ def marching_cubes(volume, level, spacing=(1., 1., 1.),
 
     See Also
     --------
-    skimage.measure.correct_mesh_orientation
+    skimage.measure.marching_cubes_lewiner
     skimage.measure.mesh_surface_area
-
+    
```

Comment:
When I wrote this, PEP 257 still called for an empty line at the end of docstrings. It no longer does, so let's just remove it.